docker/docker/src/main/java/org/arquillian/cube/docker/impl/docker/DockerClientExecutor.java

```diff
@@ -340,6 +341,16 @@ public String createContainer(String name, CubeContainer containerConfiguration)
             } else {
                 throw e;
             }
+        } catch (ConflictException e) {
```

Comment:
I thought the same but since for example the NotFoundException is also treated here, I decided to put all these automatic things in the same place.